packages/react-native-editor/jest_ui_test_environment.js

```diff
@@ -16,6 +19,9 @@ class CustomEnvironment extends JSDOMEnvironment {
 		try {
 			await super.setup();
 			this.global.editorPage = await initializeEditorPage();
+			this.global.editorPage.blockNames = blockNames;
+			this.global.E2EUtils = utils;
+			this.global.E2ETestData = testData;
```

Comment:
Is there a reason to mix camelcase and Pascal case? If the usage does not communicate something specific, it may be worth using a consistent case. 
```suggestion
			this.global.editorPage.blockNames = blockNames;
			this.global.e2eUtils = utils;
			this.global.e2eTestData = testData;
```

Reply:
Nice suggestion! I agree we should keep the consistent case, I'll update it. Thanks!

Reply:
I applied the suggestion in https://github.com/WordPress/gutenberg/pull/47845/commits/a5df02ce9e784e34ea43825a7cc235510f63d610